notebooks/official/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -1,1434 +1,1407 @@
 {
```

Comment:
<p>Train an* AutoML</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
added the article.